tools/helpers.js

```diff
@@ -0,0 +1,10 @@
+const ethers = require('ethers');
+
+export const syncTime = async function () {
```

Comment:
this is duplicated in tools/blockchain.js

Reply:
removed